source/js/carbonads.js

```diff
@@ -0,0 +1,174 @@
+export default function() {
+  var _native = (function () {
+    var _options = {}
+    var _local = window.localStorage
+    var _construct = function (e) {
+      var defaultOptions = {
+        carbonZoneKey: '',
+        fallback: '',
+        ignore: 'false',
+        placement: '',
+        prefix: 'native',
+        targetClass: 'native-ad'
+      }
+
+      if (typeof e === 'undefined') return defaultOptions
+      Object.keys(defaultOptions).forEach(function (key, index) {
+        if (typeof e[key] === 'undefined') {
+          e[key] = defaultOptions[key]
+        }
+      })
+      return e
+    }
+
+    var init = function (zone, options) {
+      _options = _construct(options)
+
+      if (isAdHidden() === true) {
+        document.addEventListener('DOMContentLoaded', function () {
+          _native.hide()
+        })
+        return
+      }
+
+      var jsonUrl = 'https://srv.buysellads.com/ads/' + zone + '.json?callback=_native_go'
+      if (_options['placement'] !== '') {
+        jsonUrl += '&segment=placement:' + _options['placement']
+      }
+      if (_options['ignore'] === 'true') {
+        jsonUrl += '&ignore=yes'
+      }
+
+      var srv = document.createElement('script')
+      srv.src = jsonUrl
+      document.getElementsByTagName('head')[0].appendChild(srv)
+    }
+
+    var carbon = function (e) {
+      var srv = document.createElement('script')
+      srv.src = '//cdn.carbonads.com/carbon.js?serve=' + e['carbonZoneKey'] + '&placement=' + e['placement']
```

Comment:
More questions for the carbon folks: why two domains?

Reply:
I think carbonads was bought by buysellads.com